src/deployment-guides/tko-saas-services.md

```diff
@@ -257,8 +257,8 @@ You can onboard Tanzu Service Mesh with or without Tanzu Mission Control.
 * (Conditionally required) An active Tanzu Mission Control subscription.  Required only if you intend to onboard a cluster to Tanzu Service Mesh via Tanzu Mission Control.
 	* Ensure that the cluster to be onboarded to Tanzu Service Mesh is attached to Tanzu Mission Control.
 * An active Tanzu Service Mesh subscription.
-* Workload cluster resource requirements:
-    - **Nodes**: At least 3 worker nodes, each with at least 3,000 milliCPU (3 CPUs) of allocatable CPU and 6 GB of allocatable memory.
+* Workload cluster resource requirements: *Please see [TSM requirements](https://docs.vmware.com/en/VMware-Tanzu-Service-Mesh/services/tanzu-service-mesh-environment-requirements-and-supported-platforms/GUID-D0B939BE-474E-4075-9A65-3D72B5B9F237.html#:~:text=Tanzu%20Service%20Mesh%20requires%20a,any%20node%20on%20the%20cluster.)*
```

Comment:
...: For workload cluster resource requirements, see [Tanzu Service Mesh Environment Requirements and Supported Platforms] (https://docs.vmware.com/en/VMware-Tanzu-Service-Mesh/services/tanzu-service-mesh-environment-requirements-and-supported-platforms/GUID-D0B939BE-474E-4075-9A65-3D72B5B9F237.html)